testsuite/features/secondary/srv_payg_ssh_connection.feature

```diff
@@ -0,0 +1,124 @@
+# Copyright (c) 2015-2021 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Listing, adding and removing ssh connection data for the payg feature
+  In order to list available ssh connections and add or remove them
+  As admin user
+  I want to use the web UI
```

Comment:
In order to use payg
As admin user
I want to list available ssh connections and add or remove them